cmd/src/bin/tikv-ctl.rs

```diff
@@ -989,6 +988,12 @@ fn main() {
         .version(version_info.as_ref())
         .long_version(version_info.as_ref())
         .setting(AppSettings::AllowExternalSubcommands)
+        .arg(
+            Arg::with_name("base-dir")
+                .long("base-dir")
+                .takes_value(true)
+                .help("Set the TiKV storage data path"),
+        )
```

Comment:
PTAL @siddontang  @iamxy

Reply:
Set or provide? We don't recommend to change the data dir

Reply:
So this is not recommended? https://github.com/tikv/tikv/blob/d654e98afab4d48f8cc2bf966ddd39c1e172ade3/cmd/src/bin/tikv-ctl.rs#L1007

Reply:
I think the `db` and `raft` is enough, no need to add this arg.